endstate_correction/simulation/base.py

```diff
@@ -101,7 +101,7 @@ def _createSystem(self, **kwargs):
     def start(self):
 
         # path where samples should be stored (will be created if it doesn't exist)
-        base = f"{self.work_dir}/equilibrium_samples/{self.name}"
+        base = f"{self.work_dir}/equilibrium_samples/"
```

Comment:
I think we might want to store the `self.name`, which will be the molecule name somewhere in the file path. The `base = f"{self.work_dir}/equilibrium_samples/{self.name}"` is the way that the trajectory is saved in the `sampling.py`